public/content/documentation/web/component/pagination-nav.md

```diff
@@ -69,12 +60,70 @@ Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/compo
 
 ### Name
 
-- Typically doesn’t have a name or description since there must be only one instance per page.
-
-## Videos
-
-- Videos go here
-<video controls>
-  <source src="media/video/native/button/buttonIosVoiceover.webm" type="video/webm">
-  Your browser does not support the video tag.
-</video>
+   - Use `aria-label="Pagination"` when there is not a visible nav title.
```

Comment:
is there a reason you switched the order of the original magenata11y content> (in the current day one it has code examples first)

Reply:
I've done it on a couple today, where "developer notes" is another heading lower in the current-day content. That way we avoid having multiple headings with the same text.